docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md

```diff
@@ -61,12 +61,12 @@ Afterwards you need to go to the Android Studio and compile, install and execute
 
 ## DataWedge profile
 
-In order to use the credential issuance part of this application (see [DeviceID tutorial](../104/zebra-iota-edge-sdk-104-tutorial.md#onboard-a-device)) you will need to set up a DataWedge profile bound to the `“org.iota.zebra.holder”` application as follows:
+In order to use the credential issuance part of this application (see [DeviceID tutorial](../zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md#onboard-a-device)) you will need to set up a DataWedge profile bound to the `“org.iota.zebra.holder”` application as follows:
 
 <p align="center">
-<img alt="Data Wedge Profile 1" src="images/image3.png" width="30%">
-<img alt="Data Wedge Profile 2" src="images/image10.png" width="30%">
-<img alt="Data Wedge Profile 3" src="images/image1.png" width="30%">
+<img alt="Data Wedge Profile 1" src="/img/zebra-iota-edge-sdk/102/image3.png" width="30%"/>
+<img alt="Data Wedge Profile 2" src="/img/zebra-iota-edge-sdk/102/image10.png" width="30%"/>
+<img alt="Data Wedge Profile 3" src="/img/zebra-iota-edge-sdk/102/image1.png" width="30%"/>
```

Comment:
Broken img embeds in markdown: https://github.com/Dr-Electron/Zebra-Iota-Edge-SDK/blob/docs/docusaurus/docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md